pipeline/src/node_properties/model_inference_properties.rs

```diff
@@ -17,6 +17,9 @@ pub struct ModelInferenceProperties {
 
     #[serde(flatten)]
     pub runtime: Option<ModelInferenceRuntime>,
+
+    /// Optional vector containing the min inference threshold value for each class.
+    pub classes_thresholds: Option<Vec<PerClassThreshold>>,
```

Comment:
Can it be just a hashmap `class_id -> min_threshold`?

Reply:
While digging in that question I found some rust profiling results that shows faster lookup using a vector than a hashmap in the case we have a small number of elements (less than 15 - which actually is the case for the majority of models classes, with the exception of yolo) 

Anyway I don't have strong opinion on that :), in the end of the day it should not be big performance difference as we want to use it mostly for full iteration and not for lookup

Reply:
The idea of optional field was handle those cases:
- if there are no `classes_thresholds` defined on console's inference node, then just load the default configuration
- otherwise if there's `classes_thresholds` create user custom config 

ofc we can just default to empty vector and later add additional check in other components checking if it's empty or not